models/Document/Tag/Image.php

```diff
@@ -421,7 +423,7 @@ public function getImage()
     }
 
     /**
-     * @param Asset\Image $image
+     * @param Asset\Image|null $image
```

Comment:
`$this` cannot be `null` right?

Reply:
The image param can be null. To reset the tag. See:
https://github.com/pimcore/pimcore/pull/7020/files#diff-3bbec05fe0a61dca677cb7e8c81411dfR803

Reply:
But the method returns `$this`

Reply:
Yes `$this` cannot be null. And the return PHPDoc is correct, right?

Reply:
says `@return Model\Document\Tag\Image|null` now

Reply:
No, it doesn't @blankse only changed the `@param` annotation, not the `@return` one.

Reply:
![image](https://user-images.githubusercontent.com/279826/91740214-448d3680-ebb3-11ea-87d1-a1236b308833.png)

Incognito says I was wrong, might have seen at an old commit or something. Sorry!